internal/runner/operation.go

```diff
@@ -130,7 +130,7 @@ func (r *Runner) executeJob(
 	//
 	// Note some operation types don't require downloaded data. These are
 	// not executed here but are executed in accept.go.
-	log.Info("executing operation", "type", fmt.Sprintf("%T", job.Operation))
```

Comment:
Removed this log arg cause its now already present in all logging statements in this func (via `log.With` in caller)